app/components/transfer/cards/TransferTypeSelect.js

```diff
@@ -31,16 +35,25 @@ export default class TransferTypeSelect extends Component<Props> {
     return (
       <div className={styles.component}>
         <div className={styles.hero}>
+          <div className={styles.instructions}>
+            <div className={styles.headerText}>
+              {intl.formatMessage(
+                messages.instruction,
+                { ticker: this.props.ticker }
+              )}
+            </div>
+            <span>
+              {intl.formatMessage(
+                messages.subInstruction,
+                { ticker: this.props.ticker }
+              )}
+              <a target="_blank" rel="noreferrer" href="https://yoroi-wallet.com/#/faq/1"> FAQ</a>
```

Comment:
Does this actually work? I don't remember why, but we used to have issues with adding `<a>` tags and so we had to use `handleExternalLinkClick` instead